lib/util.js

```diff
@@ -65,3 +78,5 @@ export const formatCode = async code => {
     singleQuote: true
   })
 }
+
+export const omit = (object, keys) => omitBy(object, (_, k) => keys.includes(k))
```

Comment:
Can you use `.indexOf(k) > -1`? `Array.includes` is not supported in all browsers yet (https://caniuse.com/#feat=array-includes)

Reply:
Ah yeah no problemo, I'm so used to just babel/polyfilling my way out of those concerns lol

Reply:
Hahaha yeah i can tell with all your class instance functions. I dig it, but... 🙂